.storybook/readmeDecorator.js

```diff
@@ -0,0 +1,18 @@
+import { makeDecorator } from "@storybook/addons";
+
+export const addReadmeToStory = makeDecorator({
+  wrapper: (storyFn, context) => {
+    try {
+      const folderPath = context.parameters.fileName.replace(/[A-Za-z.]*$/, "").replace("./src/", "");
+      context.parameters["readme"] = {
+        // https://webpack.js.org/api/module-methods/#dynamic-expressions-in-import
+        sidebar: require(`src/${folderPath}README.md`),
+      };
+    } catch (e) {
+      if (!e.message.includes("Cannot find module")) {
+        console.log(e);
```

Comment:
Error handling means, that you handle one type of error in a special way. (In this case we want to be silent about "Cannot find module").

This means that you need to throw all other types of error, so the parent module (and all it's parent modules) will try to handle this error or throw it.

console.logging the error is not throwing it back.